channels/al.m3u

```diff
@@ -27,7 +27,7 @@ http://us.bestvideostreaming.is/8821/playlist.m3u8
 http://albuk.dyndns.tv:1935/albuk/albuk.stream/playlist.m3u8
 #EXTINF:-1 tvg-id="ALPO.al" tvg-country="AL" tvg-language="Albanian" tvg-logo="https://i.imgur.com/00hiXCL.png" group-title="",ALPO (720p) [Not 24/7]
 https://5d00db0e0fcd5.streamlock.net/7236/7236/playlist.m3u8
-#EXTINF:-1 tvg-id="Alsat.al" tvg-country="AL" tvg-language="Albanian" tvg-logo="" group-title="",Alsat (1080p) [Not 24/7]
+#EXTINF:-1 tvg-id="Alsat.al" tvg-country="AL" tvg-language="Albanian" tvg-logo="" group-title="",Alsat (720p) [Not 24/7]
 http://93.157.62.180/AlsatM/index.m3u8
 #EXTINF:-1 tvg-id="ARTAHD.al" tvg-country="AL" tvg-language="Albanian" tvg-logo="" group-title="",ARTA HD (720p) [Geo-blocked]
 http://us.bestvideostreaming.is/8745/playlist.m3u8
```

Comment:
**_http://us.bestvideostreaming_** is not for albanian viewers. Please upload links for Albania.